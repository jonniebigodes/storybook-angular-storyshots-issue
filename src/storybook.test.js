/* import * as path from 'path';
import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'angular',
  configPath: path.join(__dirname, '../.storybook'),
  test: multiSnapshotWithOptions(),
}); */
import registerRequireContextHook from 'babel-plugin-require-context-hook/register';
import initStoryshots from '@storybook/addon-storyshots';

registerRequireContextHook();
initStoryshots();