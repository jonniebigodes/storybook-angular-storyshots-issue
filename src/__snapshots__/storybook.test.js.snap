// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Task archived 1`] = `
<storybook-dynamic-app-root
  cfr={[Function CodegenComponentFactoryResolver]}
  data={[Function Object]}
  target={[Function ViewContainerRef_]}
>
  <app-task>
    <div
      class="list-item TASK_ARCHIVED"
    >
      <label
        class="checkbox"
      >
        <input
          checked=""
          disabled="true"
          name="checked"
          type="checkbox"
        />
        <span
          class="checkbox-custom"
        />
      </label>
      <div
        class="title"
      >
        <input
          ng-reflect-ng-style="[object Object]"
          placeholder="Input title"
          readonly="true"
          style="text-overflow: ellipsis;"
          type="text"
        />
      </div>
      <div
        class="actions"
      >
        
      </div>
    </div>
  </app-task>
</storybook-dynamic-app-root>
`;

exports[`Storyshots Task default 1`] = `
<storybook-dynamic-app-root
  cfr={[Function CodegenComponentFactoryResolver]}
  data={[Function Object]}
  target={[Function ViewContainerRef_]}
>
  <app-task>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="true"
          name="checked"
          type="checkbox"
        />
        <span
          class="checkbox-custom"
        />
      </label>
      <div
        class="title"
      >
        <input
          ng-reflect-ng-style="[object Object]"
          placeholder="Input title"
          readonly="true"
          style="text-overflow: ellipsis;"
          type="text"
        />
      </div>
      <div
        class="actions"
      >
        
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
  </app-task>
</storybook-dynamic-app-root>
`;

exports[`Storyshots Task longTitle 1`] = `
<storybook-dynamic-app-root
  cfr={[Function CodegenComponentFactoryResolver]}
  data={[Function Object]}
  target={[Function ViewContainerRef_]}
>
  <app-task>
    <div
      class="list-item TASK_INBOX"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="true"
          name="checked"
          type="checkbox"
        />
        <span
          class="checkbox-custom"
        />
      </label>
      <div
        class="title"
      >
        <input
          ng-reflect-ng-style="[object Object]"
          placeholder="Input title"
          readonly="true"
          style="text-overflow: ellipsis;"
          type="text"
        />
      </div>
      <div
        class="actions"
      >
        
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
  </app-task>
</storybook-dynamic-app-root>
`;

exports[`Storyshots Task pinned 1`] = `
<storybook-dynamic-app-root
  cfr={[Function CodegenComponentFactoryResolver]}
  data={[Function Object]}
  target={[Function ViewContainerRef_]}
>
  <app-task>
    <div
      class="list-item TASK_PINNED"
    >
      <label
        class="checkbox"
      >
        <input
          disabled="true"
          name="checked"
          type="checkbox"
        />
        <span
          class="checkbox-custom"
        />
      </label>
      <div
        class="title"
      >
        <input
          ng-reflect-ng-style="[object Object]"
          placeholder="Input title"
          readonly="true"
          style="text-overflow: ellipsis;"
          type="text"
        />
      </div>
      <div
        class="actions"
      >
        
        <a>
          <span
            class="icon-star"
          />
        </a>
      </div>
    </div>
  </app-task>
</storybook-dynamic-app-root>
`;
